### DISCUSSION TITLE
Schedule Switching Profiles

### DISCUSSION THREAD

**Discussion by ohbmah28** [Rem0o/FanControl.Releases#468] [Ideas]
Let's say I have 2 profiles, one with all fan control turned off, and the other with the controls turned on and set to the lowest RPM. I can then schedule the profile with the lowest RPM to be turned on at 12 am for example, and then it switches back to the normal profile at 8 am. This would be useful for people who keep their PCs up 24/7 in their room.
Btw love the software!

**Comment by Rem0o** [comment]
What's the use case of turning everything off during the day?

**Reply by ohbmah28** [reply]
Sorry about my wording, I made it confusing. I use my PC as a server and I keep it running 24/7, I'd like to schedule the PC to turn off the fans at midnight so I can sleep without the fan noise, and then schedule it to turn the fans back on at my regular wake up time

**Reply by Rem0o** [reply]
Why don't you keep it quiet all the time? If it can live at night with low fan speeds, it can also during the day. Do you have fan curves based on temperature at all?

**Reply by ohbmah28** [reply]
I also game on the machine so having the fans turned off all the time isn’t very ideal, and switching profiles every time I launch a game isn’t very convenient

**Reply by Rem0o** [reply]
Do you have any fan curves at all based on temperatures? It would cover your gaming with higher speeds, and server load with lower speeds.

**Reply by ohbmah28** [reply]
I haven’t looked into fan curves, but it sounds like it would work for me. Thanks

**Reply by RaphelMadeira** [reply]
@Rem0o  I REALLY like the idea of profiles changing automatically based on Windows time. Currently, I use two different profiles, the "Auto (BETA)" which is quite noisy until 10 pm, and after that, I manually change it to a "Graph" fan curve created by me to be much quieter, as the nights are cooler and therefore not so much ventilation is needed inside the CPU.

The idea of automatically changing profiles based on the Windows clock time would be very interesting because several times I turn on the computer the next morning and forget to change the profile I had used the night before (silent) "Graph " fan curve to the "Auto (BETA)" daytime profile that better cools the inside of the CPU. And I end up scaring myself with the temperature of the processor.

The app is perfect, but a profile according to the computer's time would make it even better! Thanks for your work!

**Reply by Rem0o** [reply]
Well, now you can do exactly that with scheduled tasks.

Simply call FanControl.exe --config theconfigyouwanthere.json

**Reply by RaphelMadeira** [reply]
@Rem0o Thank you very much for your answer, I made a profile here just to find out if something like this already existed and to thank you, I'm going to search out and study more about .json files to understand more about it.

Thanks a lot, have a great week!

**Reply by Rem0o** [reply]
json files are just your saved configs in FanControl.  When you save as => a name, it will create "a name.json".

**Reply by Y4m4k** [reply]
Hi all - would it be possible to have profiles switch set at certain GPU/CPU temps? For example, I have a "quiet.jason" profile, however, when I start gaming and the average temperature increases by X amount of degrees, it will automatically switch to "gaming.jason" profile. Then when GPU/CPU decreases by X amount of degrees, it will go back to "quiet.jason". It would nice functionality instead of manually setting every time one games.
